src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporter.cs

```diff
@@ -75,15 +75,25 @@ internal OtlpTraceExporter(OtlpExporterOptions options, OtlpCollector.TraceServi
             }
             else
             {
-                if (options.Endpoint.Scheme == Uri.UriSchemeHttps)
+                if (options.Endpoint.Scheme != Uri.UriSchemeHttp && options.Endpoint.Scheme != Uri.UriSchemeHttps)
                 {
-                    throw new NotSupportedException("Https Endpoint is not supported.");
+                    throw new NotSupportedException($"Endpoint URI scheme ({options.Endpoint.Scheme}) is not supported.");
                 }
 
 #if NETSTANDARD2_1
                 this.channel = GrpcChannel.ForAddress(options.Endpoint);
 #else
-                this.channel = new Channel(options.Endpoint.Authority, ChannelCredentials.Insecure);
+                ChannelCredentials channelCredentials;
+                if (options.Endpoint.Scheme == Uri.UriSchemeHttps)
+                {
+                    channelCredentials = new SslCredentials();
```

Comment:
So those that want a secure channel I guess would specify the cert path via the environment variable? If so this seems like it would work for the simple case. I hope the additional tests prove this out and this gets into 1.0!

Reply:
https://grpc.github.io/grpc/csharp/api/Grpc.Core.SslCredentials.html#Grpc_Core_SslCredentials__ctor
We shouldn't recommend customers to rely on this env variable. Its an implementation detail that it works, so they shouldn't bet on it. 

Instead, the actual environment variable, as per otel spec should be added in 1.1.

Reply:
Make sense.